cypress/integration/vesselCommunications.spec.ts

```diff
@@ -1,57 +1,231 @@
-import { givenIHaveSelected, whenIClickContinue } from "./common.spec";
-
-describe("As a beacon owner, I want to register my communication details so SAR can contact me in an emergency", () => {
+import {
+  andIClickContinue,
+  finallyIClear,
+  finallyIUncheck,
+  givenIAmAt,
+  givenIHaveSelected,
+  iCanClickTheBackLinkToGoToPreviousPage,
+  thenIShouldSeeAnErrorMessageThatContains,
+  thenIShouldSeeAnErrorSummaryLinkThatContains,
+  thenMyFocusMovesTo,
+  thenTheUrlShouldContain,
+  whenIClickContinue,
+  whenIClickOnTheErrorSummaryLinkContainingText,
+  whenIType,
+} from "./common.spec";
+
+describe("As a beacon owner and maritime pleasure vessel user", () => {
   const pageUrl = "/register-a-beacon/vessel-communications";
 
   beforeEach(() => {
-    givenIAmOnTheVesselCommunicationsPage();
+    givenIAmAt(pageUrl);
   });
 
-  it("requires an MMSI number if the fixed VHF checkbox is selected", () => {
-    givenIHaveSelected("#fixedVhfRadio");
-    andIHaveLeftTheRelevantTextInputBlank();
-
-    whenIClickContinue();
+  describe("the Fixed VHF/DSC radio option", () => {
+    it("requires an MMSI number if the fixed VHF/DSC radio checkbox is selected", () => {
+      const expectedErrorMessage = ["We need", "fixed VHF"];
+      givenIHaveSelected("#fixedVhfRadio");
+
+      whenIType(" ", "fixedVhfRadioInput");
+      whenIClickContinue();
+      thenIShouldSeeAnErrorSummaryLinkThatContains(...expectedErrorMessage);
+      thenIShouldSeeAnErrorMessageThatContains(...expectedErrorMessage);
+      whenIClickOnTheErrorSummaryLinkContainingText(...expectedErrorMessage);
+      thenMyFocusMovesTo("fixedVhfRadioInput");
+
+      finallyIClear("#fixedVhfRadioInput");
+      finallyIUncheck("#fixedVhfRadio");
+    });
+
+    it("requires the fixed MMSI number to be 9 characters long", () => {
+      const expectedErrorMessage = ["MMSI number", "nine digits long"];
+      givenIHaveSelected("#fixedVhfRadio");
+
+      whenIType("012345678910", "fixedVhfRadioInput");
+      andIClickContinue();
+
+      thenIShouldSeeAnErrorSummaryLinkThatContains(...expectedErrorMessage);
+      thenIShouldSeeAnErrorMessageThatContains(...expectedErrorMessage);
+      whenIClickOnTheErrorSummaryLinkContainingText(...expectedErrorMessage);
+      thenMyFocusMovesTo("fixedVhfRadioInput");
+
+      finallyIClear("#fixedVhfRadioInput");
+      finallyIUncheck("#fixedVhfRadio");
```

Comment:
Teardown step to avoid the expensive operation of either reloading the page or `cy.get()`ting and `cy.uncheck()`/`cy.clear()`ing every form input.  (There is no single `cy.form().clear()` unfortunately)

Reply:
Removed due to state bleeding between tests.  Chose to reload page in the `beforeEach` hook instead.